### ISSUE TITLE
are there any power ups in the game?

### ISSUE THREAD

**Issue by JeanFenech** [mabela25942/MarlonAbela-Paper-Catcher#3]
can you boost with power ups in the game?

**Comment by mabela25942** [comment]
yes there are 2 power ups which are a gold coin that grants the user an increase in points and there is also a heart which grants the user an extra life.
